abis_mapping/templates/dwc_mvp/mapping.py

```diff
@@ -105,6 +106,10 @@ def apply_mapping(
             onerror="raise",  # Raise errors, it should already be valid here
         )
 
+        # Infer Statistics and Count Number of Rows
+        resource.infer(stats=True)
```

Comment:
How safe is this `infer` method? Is it part of frictionless?

Reply:
@kehh Yes, from `frictionless`: https://framework.frictionlessdata.io/docs/guides/framework/resource-guide/#stats